backtrace-library/src/main/java/backtraceio/library/common/BacktraceStringHelper.java

```diff
@@ -0,0 +1,11 @@
+package backtraceio.library.common;
+
+public class BacktraceStringHelper {
+    public static boolean isNullOrEmpty(String input) {
+        return (input == null || input.trim().isEmpty());
+    }
+
+    public static boolean isObjectValidString(Object input) {
+        return (input != null && !input.toString().trim().isEmpty());
```

Comment:
it's ... really bad idea - every class has toString method that will return something - input.toString().trim().isEmpty() will be always true. For example: isObjectValidString(new Person()) should return true - is it expected? 
if it's not - we need to check if object is type string.

Reply:
I think it's okay because we use the same logic for converting Object to String in `convertAttributes` function. We just trust `Object.toString()` method.
https://github.com/backtrace-labs/backtrace-android/blob/dcf0eb97675d693c5aff0f733d407197e9dad168/backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceAttributes.java#L188

Reply:
@konraddysput I think we only need to check if object is not null. GSON will take care of serializing this object to a string later anyway so we don't need to worry about it. I will remove unnecessary `Object.toString()` call.

Reply:
@konraddysput sounds good, we will only do string empty check if object is of type string. Otherwise if it's a regular object, GSON will take care of serializing the object and we just need to check it's not null.